### DISCUSSION TITLE
Github Copilot for business billing on prepaid EA Azure Subscripción

### DISCUSSION THREAD

**Discussion by Kelvfimer** [community/community#57187] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Please, I would like to know when Github Copilot will support billing on prepaid EA Azure Subscriptions. I know that for EA MACCs contract is on roadmap but for prepaid EA contract there is no info at all.

thx

**Comment by davecheney** [comment, marked as answer]
@Kelvfimer please contact your sales rep or support engineer, they can assist you.
